pymess/backend/dialer/daktela.py

```diff
@@ -91,6 +100,9 @@ def publish_message(self, message):
                 },
                 'action': 5,
             }
+            custom_fields = message.extra_data.get('custom_fields')
+            if custom_fields:
+                payload['customFields'].update(**custom_fields)
```

Comment:
k čemu je toto?

Reply:
už jsem našel. custom fields které posíláme do daktely při vytvoření zprávy, je takové divné, že se to bere z extra data, do kterých se ukládají výstupní data

Reply:
ale čert to vem